contracts/src/v0.8/feeds/test/DualAggregator.t.sol

```diff
@@ -454,12 +454,14 @@ contract GetRequesterAccessController is DualAggregatorBaseTest {
 // TODO: determine if we need this method still
 contract RequestNewRound is ConfiguredDualAggregatorBaseTest {}
 
-contract Trasmit is ConfiguredDualAggregatorBaseTest {
+contract Transmit is ConfiguredDualAggregatorBaseTest {
   uint32 epoch = 0;
   uint32 round = 0;
```

Comment:
I think these can be deleted?

Reply:
For me they could be removed, but I'm not sure.

I see we're using them but never updating the value.

@KMontag42, since you added this values, do you have any objections to removing these?